crash-handler/build.gradle.kts

```diff
@@ -9,7 +9,7 @@ plugins {
 apply(from = "../config/quality.gradle")
 
 android {
-    namespace = "org.odk.collect.crash_handler"
+    namespace = "org.odk.collect.crashhandler"
```

Comment:
So we can't use dashes here?

Reply:
We can for the module name, but not the Java package annoyingly.